README.md

```diff
@@ -3,3 +3,14 @@
 # Bitwarden JavaScript Library
 
 Common code referenced across Bitwarden JavaScript projects.
+
+## Requirements
+
+* Git
+* node-gyp
+
+### Windows
+
+* *Microsoft Build Tools 2015* in Visual Studio Installer
```

Comment:
Do we know if 2017 build tools are also required?

Reply:
I have VS 2019 installed in a VM with only 2015 and not 2017.